src/targets/browser/serviceWorkerRegistration.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://cra.link/PWA
+
+const isLocalhost = Boolean(
+  window.location.hostname === 'localhost' ||
```

Comment:
It can not work for us, right? It should be something like: 

window.location.hostname.endsWith('cozy.localhost') || window.location.hostname.endsWith('cozy.tools')

No?

Reply:
Indeed
As today we just need a user to be able to refresh their PWA, I also lightened the code